### DISCUSSION TITLE
Customization of overall theme color

### DISCUSSION THREAD

**Discussion by elliottmckee** [powerplanner/powerplannerapps#97] [Ideas]
I have been a huge fan of PowerPlanner (mainly for aesthetic reasons) since I stumbled across it. 

It would be very nice to be able to change the default blue of the top and sidebar to a custom color, similar to that of classes. This would go a long way towards being able to make cohesive themes (e.g. i've personally been on a big Dracula kick lately)

I might poke around at trying to do this myself if I get the chance, as it seems relatively straightforward to implement (at least manually overriding it), but that's dependent on me finding the time. 

Appreciate this planner- it has been with me throughout the majority of my college career, and I plan on using it for a while to come :)

**Comment by Charlseao** [comment]
Hi! Did you ever happen to figure out how to change the color? I would also love to change the blue color out.  Thanks!
